### PR TITLE
dev to master

### PR DESCRIPTION
**Pull request by ZofiaRudjord**

### CODE REVIEW

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py [hunk 1]
```diff
@@ -332,6 +332,13 @@ def loggers_variables() -> List[VariablesCreate]:
             "variablecode": "TempCond_Avg",
             "nodatavalue": -9999,
         },
+        {
+            "variabletypecv": "Chemistry",
+            "variablenamecv": "Temperature, sensor",
+            "variabledefinition": "Temperature measured by Ph sensor",
```
Comment (haakondr) [inline src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:338]: question: does it make sense to differentiate temperature variables by how they are measured? Does temp measured by a ph sensor differ from another sensor? I may not be seeing the full picture here

**Review by haakondr — APPROVED** [review]
lgtm

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py (hunk 1 above)
Comment (ZofiaRudjord) [reply src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:338]: It is important for operation people (like Uta). I opened an issue about it in nivacloud-loggers: https://github.com/NIVANorge/nivacloud-loggers/issues/39

Comment (ZofiaRudjord) [reply src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:338]: But this is where cv' come in handy. We can always query all time series's which measure temperature by
putting a constraint on "variablenamecv"

**Pull request merged**